### DISCUSSION TITLE
Testing and Playing around with Wiki.js V3

### DISCUSSION THREAD

**Discussion by jarrodlovas** [requarks/wiki#5459] [General]
Hey Everyone,

I've got wikijs vega branch working because I want to start exploring and tinkering around for fun.
It gets to this stage, and nothing loads.
![image](https://user-images.githubusercontent.com/47882472/178100585-9b5fa9ca-69af-499b-938b-bf6169c7df4c.png)

This is what is displayed in the browser.
![image](https://user-images.githubusercontent.com/47882472/178100606-12753c17-3ae8-4337-a3a9-19614af254fc.png)

Is there something that needs to be done for end-users to start setting up and thinking with 3.0

**Comment by cdn0x12** [comment]
You need to build the assets manually in both the /ux folder and the root folder(assets-legacy).
As [the blog post](https://blog.js.wiki/news/2022/the-road-to-the-wiki-js-3-beta) says, the vega branch is not yet in a usable state. Even if you build the assets, configures your database and launches the project, it will not function as you expect. You will just see the "Unauthorized" error page and could do nothing.
I think one of the reason is that the GraphQL backend is being reworked to bring all queries and mutations at the root level. That requires a large part of the frontend's logic to be rewritten.

A good news: If you update your clone to [commit#82ebac2d](https://github.com/requarks/wiki/commit/82ebac2dd6acb9f4ff8d18569db6974730051404) and build it, you will see the new style of the 403/404 page and login page by just clicking your mouse.
![image](https://user-images.githubusercontent.com/46210502/184684704-e15d309d-7391-452d-a68f-2dc8337cf3ef.png)
![image](https://user-images.githubusercontent.com/46210502/184684762-dc7d0774-4ec5-44ef-8a17-e4e8488997ce.png)
![image](https://user-images.githubusercontent.com/46210502/184684825-17a14800-ea6a-44ea-b188-fc3fe0e3352b.png)

**Comment by fridemar** [comment]
Is there a real wiki.js, with a sandbox page, to try basic editing.

**Comment by fridemar** [comment]
Is there a ready made installation program for running a local wiki on Osx Macbook.

**Reply by BPowell76** [reply]
If you want to mess with version 2.5, you can follow the steps on their [wiki](https://docs.requarks.io/install)
